lib/inc/hocon/config_exception.hpp

```diff
@@ -12,10 +16,10 @@ namespace hocon {
      */
     struct config_exception : public std::runtime_error {
         config_exception(config_origin const& origin, std::string const& message) :
-                runtime_error(origin.description() + ": " + message) { }
+                runtime_error(_("{1}: {2}", origin.description(), message)) { }
```

Comment:
Origin is user-specified (usually a file name). Message appears to be translated at the calls to the config_exception constructor. See line 33 for 1 example.